src/sasjs-servicepack/index.js

```diff
@@ -0,0 +1,88 @@
+import { servicePackDeploy } from './deploy'
+
+import chalk from 'chalk'
+
+export async function processServicepack(commandLine) {
+  const command = commandLine[1]
+  const commands = {
+    deploy: 'deploy'
+  }
+
+  if (!commands.hasOwnProperty(command)) {
+    console.log(
+      chalk.redBright(
+        `Not supported context command. Supported commands are:\n${Object.keys(
+          commands
+        ).join('\n')}`
+      )
+    )
+
+    return
+  }
+
+  const commandExample =
+    'sasjs servicepack <command> --source ../viyadeploy.json --target targetName'
+
+  const getIsForced = () => {
```

Comment:
If this, and the functions below were outside the scope of the main function, they can be easily unit tested.

Reply:
aah yes - these could be in utils right?

Reply:
Yes, or at least they could be separate exported functions in the same file, which take the `commandLine` variable as a parameter, so they can be tested in isolation.